theme/theme.inc

```diff
@@ -10,14 +10,11 @@
  */
 function islandora_pdfjs_preprocess_islandora_pdfjs(array &$variables) {
   $fedora_object = $variables['fedora_object'];
+  $dsid = (isset($variables['dsid']) ? $variables['dsid'] : NULL);
 
-  if (isset($fedora_object['OBJ']) && $fedora_object['OBJ']->mimetype == 'application/pdf') {
-    $dsid = 'OBJ';
+  if ($dsid) {
+    $file_url = url("islandora/object/{$fedora_object->id}/datastream/$dsid/view", array('absolute' => TRUE));
+    $variables['file_url'] = $file_url;
+    $variables['viewer_url'] = url(libraries_get_path('pdfjs') . '/web/viewer.html', array('absolute' => TRUE, 'query' => array('file' => $file_url)));
```

Comment:
Looks like this was an issue before... Stuff going directly to files should use `file_create_url()`, as `url()` can add language prefixes and multisite components and the like (when not used for "external" URLs--for stuff going through Drupal's menu routing). 

To be able to tack on the query params, I'm thinking something like:
```php
$lib_path = libraries_get_path('pdfjs');
$variables['viewer_url'] = url(file_create_url("$lib_path/web/viewer.html"), array(
  'external' => TRUE,
  'query' => array(
    'file' => $file_url,
  ),
));
```